spec/sample_feeds/json_feed.json

```diff
@@ -6,6 +6,8 @@
   "feed_url": "http://inessential.com/feed.json",
   "user_comment": "This feed allows you to read the posts from this site in any feed reader that supports the JSON Feed format. To add this feed to your reader, copy the following URL — http://inessential.com/feed.json — and add it your reader.",
   "favicon": "http://inessential.com/favicon.ico",
+  "icon": "http://inessential.com/icon.png",
+  "language": "en-US",
```

Comment:
I added those, although those attributes do _not_ exist in the actual feed, and the `icon` does not even exist. But I thought for testing purposes, this is still ok. Hope that’s the case.